hack/verify.sh

```diff
@@ -9,3 +9,4 @@ ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 xargs "$ROOT/hack/lib/verify-codegen.sh" < "${ROOT}/hack/lib/kube-versions.txt"
 "$ROOT/hack/module.sh" lint
+"$ROOT/hack/lib/verify-cli-reference-docs.sh"
```

Comment:
Just FYI this `verify.sh` isn't 100% the same as the one that runs in CI, so you'll need to add this line to the Concourse task as well.

Reply:
Discussed in Slack: https://kubernetes.slack.com/archives/C01BW364RJA/p1616426305147400. I'll add this to the `run-verify-lint` CI job.

Reply:
I'll add this to the `run-verify-lint` CI job after this PR is merged.